src/SourceBuild/content/eng/bootstrap/buildBootstrapPreviouslySB.csproj

```diff
@@ -153,7 +157,7 @@
 
     <!-- Repack tarball with new bootstrap name -->
     <Message Text="  Repacking tarball to $(NewTarballName)" Importance="High" />
-    <Exec Command="tar --numeric-owner -czf $(NewTarballName) *.nupkg *.props SourceBuildReferencePackages/" WorkingDirectory="$(UnpackedTarPath)" />
+    <Exec Command="tar --numeric-owner -czf $(NewTarballName) *.nupkg *.props *.xml SourceBuildReferencePackages/" WorkingDirectory="$(UnpackedTarPath)" />
```

Comment:
Should the .xml glob also be more specific?

Reply:
I wanted to follow the existing pattern, so I did the same thing we do with PackageVersions.props (use a glob). I'd rather leave it that way and clean up both in the future than have one do one thing and another do something different.